orangewidget/gui.py

```diff
@@ -1113,6 +1124,12 @@ def button(widget, master, label, callback=None, width=None, height=None,
     :rtype: QPushButton
     """
     button = buttonType(widget)
+    if button.style().metaObject().className() == 'QMacStyle':
+        btnpaddingbox = vBox(widget, margin=0, spacing=0)
+        separator(btnpaddingbox, 0, 4)  # lines up with a WA_LayoutUsesWidgetRect checkbox
+        button.outer_box = btnpaddingbox
```

Comment:
Why would you assume anything related to checkbox here?

Reply:
Autocommit is a button next to a checkbox. Generally you want all buttons in a row (QHBoxLayout) at the same height, so it's set globally.

Without this, still keeping `Qt.WA_LayoutUsesWidgetRect` will misalign the checkbox and button.